src/Apps/Collect2/Routes/Collection/Components/CollectionsHubRails/ArtistSeriesRail/index.tsx

```diff
@@ -95,7 +88,21 @@ export const ArtistSeriesRail: React.FC<ArtistSeriesRailProps> = ({
             </ArrowContainer>
           )
         }}
+        onArrowClick={() => trackArrowClick()}
       />
+    )
+  }
+
+  return (
+    <Content mt={2} py={3}>
+      <Serif size="5" mb={1}>
+        Trending Artist Series
+      </Serif>
+      <Media lessThan="lg">
+        {renderCarousel({ xs: true, sm: true, md: true })}
```

Comment:
This threw me off at first; the line reads like we are saying that the current breakpoint is _always_ `xs`, `sm`, _and_ `md`. 

I think I understand how `renderCarousel` is using these values, but I wonder what it would look like if we passed the breakpoint-based computed values _into_ `renderCarousel` instead of passing the breakpoints themselves? Something like this:

```
<Media lessThan="lg">
  {renderCarousel({
    groupCells: 1, 
    showArrows: members.length <= 4 && !sd.IS_MOBILE
  })
</Media>
<Media at="lg">
  {renderCarousel({
    groupCells: 4, 
    showArrows: members.length <= 4 && !sd.IS_MOBILE
  })
</Media>
<Media at="lg">
  {renderCarousel({
    groupCells: 4, 
    showArrows: false
  })
</Media>
```

And then `renderCarousel` itself would become a little simpler, and just display based on the computed values passed in.

This, to me, feels like I can tell more quickly what the differences are between the different breakpoints, because all the decisions are in one place.

Reply:
I also think the names `showArrows` in my example and `showArrowChecker` in the actual code are slightly misleading. They imply that this flag decides 100% if the arrows are showing...but it seems to be more complicated than that. I'm not sure what it would be, but is there a more direct name?